test/localintegrationrunner.html

```diff
@@ -3,13 +3,13 @@
     <meta charset="utf-8">
     <!-- encoding must be set for mocha's special characters to render properly -->
     <link rel="stylesheet" href="../node_modules/mocha/mocha.css" />
-    <script src="./vendor/lodash.min.js"></script>
   </head>
   <body>
     <div id="deb"></div>
     <div id="mocha"></div>
     <script src="../node_modules/mocha/mocha.js"></script>
     <script src="hacks/phantomhacks.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.20/lodash.min.js" integrity="sha512-90vH1Z83AJY9DmlWa8WkjkV79yfS2n2Oxhsi2dZbIv0nC4E6m5AbH8Nh156kkM7JePmqD6tcZsfad1ueoaovww==" crossorigin="anonymous"></script>
```

Comment:
why are we doing this, especially given that `lodash` is still a dependency of the package?

Reply:
nathan and i spoke about this. i feel strongly that the experience for using `ripple-lib` in browser vs node should not be different. for that reason, `lodash` should be included (or not) regardless of target. the size savings are not worth the confusion to users who just want to pick this lib up and use it, especially considering that `ripple-lib` is already over 5x smaller than `web3.js`.

Reply:
If we're actually packaging lodash now, then the custom lodash build might be more worth it

Reply:
Yeah, I still think we wanna rip out lodash in v2... 

I'm ok w/ leaving it like this for this PR, thoughts?

Reply:
Personally, I agree with Nate:
1. I'm ok w/ leaving it like this for this PR
2. I want to get rid of lodash in v2. Removing it entirely is a significant effort, but very worthwhile, IMHO

Just my opinion though: it's ok if y'all decide otherwise

Reply:
sorry - i think we're talking about two different things.

1) if we want to remove lodash for v2 that's great
2) but as long as it's in v1 i think we should include it in the dist so that web users don't have to know to do this.

the second bit is outside the scope of this PR, but I'd love to see a ticket for v1 to add it back in for browser dist, unless/until is completely removed as a dependency for all platforms